Neos.Media/Classes/Domain/Model/AssetSource/AssetNotFoundException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+namespace Neos\Media\Domain\Model\AssetSource;
+
+/*
+ * This file is part of the Neos.Media package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+  *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Exception;
```

Comment:
Hey wasn't one of my learnings from the Con, that introducing Neos\Flow\Exception was a bad idea and that one should not use it :) Which talk was ist @kitsunet / @kdambekalns?

Reply:
I extend the exception because then I have a proper HTTP status code (in this case 404). Would be better if Flow provided an interface for that, but currently it doesn't.

I didn't know that it was bad practice …